examples/graphene_apps/python_workloads/Dockerfile

```diff
@@ -20,11 +20,20 @@ RUN ln -sf /usr/bin/python3 /usr/bin/python
 
 WORKDIR /home/pythonworker
 
-COPY . .
-COPY ./ov_workload/ov_workload_config.toml /home/pythonworker/ov_workload_config.toml
+COPY examples/graphene_apps/python_workloads .
```

Comment:
This dockerfile was created to build Graphene Avalon python worker and not the workload. 
Python worker needs to be moved to $TCF_HOME/tc/graphene/python_worker. The Docker file also needs to be moved in same location as python worker

Reply:
Ok, will move docker file to $TCF_HOME/tc/graphene/python_worker